content/automate/ManageIQ/System/Request.class/__methods__/dump_root.rb

```diff
@@ -0,0 +1,12 @@
+#
+# Description: Log all attributes stored in the $evm.root hash.
+#
+
+def dump_root
```

Comment:
@billfitzgerald0120 to aid in testing we need to be able to inject dependencies. we won't have a $evm when we test 
so this might be a better way of accomplishing the same thing
```
def dump_root(handle = $evm)
    handle.log("info", "Root:<handle.root> Attributes - Begin")
    handle.root.attributes.sort.each { |k, v| handle.log("info", "  Attribute - #{k}: #{v}") }
    handle.log("info", "Root:<handle.root> Attributes - End")
    handle.log("info", "")
 end
```